test/parallel/test-crypto-random.js

```diff
@@ -487,7 +487,17 @@ common.expectsError(
   }
 );
 
+common.expectsError(
+  () => crypto.randomBytes(1, null),
```

Comment:
Could this also be put inside the `forEach` loop? Or have a new `forEach` loop around it? Then we're good to go. :)

Reply:
I moved it to inside the new forEach loop to omit undefined in test case.